tasks/collect-collectors-data/README.md

```diff
@@ -0,0 +1,21 @@
+# collect-collectors-data
+
+Tekton task to collect the information related to collectors.
+
+The purpose of this task is to collect all the data and supply it to the other task in the pipeline by creating
```

Comment:
"to the other task"? which one? probably you want "to other tasks"?

Reply:
Absolutely right.